db/migrate/20221028081629_create_coments.rb

```diff
@@ -0,0 +1,10 @@
+class CreateComents < ActiveRecord::Migration[7.0]
+  def change
+    create_table :coments do |t|
```

Comment:
- [ ] I think this table name should be `comments`, not `coments`. Kindly fix this table name.

- [ ] Remember to rename the related model and file name as well: **`Comment`**